### PR TITLE
APPS-4420 - Preserve route and don't rewrite request

### PR DESCRIPTION
**Pull request by ElanHasson**

### CODE REVIEW

**Comment by scotchneat** [comment]
!deploy

File: specification/resources/apps/models/app_route_spec.yml [hunk 1]
```diff
@@ -6,3 +6,7 @@ properties:
       all components within an app.
     type: string
     example: /api
+  preserve_path_prefix:
+    description:  Boolean. An optional flag to preserve the path that is forwarded to the backend service. By default, the HTTP request path will be trimmed from the left when forwarded to the component. For example, a component with `path=/api` will have requests to `/api/list` trimmed to `/list`. If this value is `true`, the path will remain `/api/list`.
```
Comment (scotchneat) [inline specification/resources/apps/models/app_route_spec.yml:10]: Does `Boolean` need to be specified in the description if the property has `type: boolean`?

```suggestion
    description: An optional flag to preserve the path that is forwarded to the backend service. By default, the HTTP request path will be trimmed from the left when forwarded to the component. For example, a component with `path=/api` will have requests to `/api/list` trimmed to `/list`. If this value is `true`, the path will remain `/api/list`.
```

Comment (ElanHasson) [reply specification/resources/apps/models/app_route_spec.yml:10]: Sorry, just saw this now.

**Review by scotchneat — APPROVED** [review]
LGTM. Thanks @ElanHasson!

**Pull request merged**